po/fr.po

```diff
@@ -233,31 +228,30 @@ msgstr ""
 "newsboat : rechargement terminé, %f fils à lire (%n articles à lire au total)"
 
 #: src/configcontainer.cpp:272
-#, fuzzy
 msgid ""
 "%N %V - Articles in feed '%T' (%u unread, %t total)%?F? matching filter `"
 "%F'&? - %U"
-msgstr "%N %V - Articles dans le fil '%T' (%u à lire, %t au total) - %U"
+msgstr "%N %V - Articles dans le fil '%T' (%u non lus, %t au total)"
+"%?F? correspondant au filtre `%F'&? - %U"
 
 #: src/configcontainer.cpp:278
 msgid "%N %V - Dialogs"
 msgstr "%N %V - Vues"
 
 #: src/configcontainer.cpp:281
-#, fuzzy
 msgid ""
 "%N %V - %?F?Feeds&Your feeds? (%u unread, %t total)%?F? matching filter `"
 "%F'&?%?T? - tag `%T'&?"
-msgstr "%N %V - Vos fils (%u à lire, %t au total)%?T? - étiquette `%T'&?"
+msgstr "%N %V - Vos fils (%u non lus, %t au total)%?F? correspondant au"
```

Comment:
There should be a space here, or we'll get "aufiltre" when the lines are concatenated.